doc/requirements.txt

```diff
@@ -30,6 +30,6 @@ lxml_html_clean
 
 # Pre-install PL development wheels
 --extra-index-url https://test.pypi.org/simple/
-pennylane-lightning-kokkos==0.40.0
-pennylane-lightning==0.40.0
-pennylane==0.41.0-dev20
+pennylane-lightning-kokkos==0.41.0-dev22
+pennylane-lightning==0.41.0-dev22
```

Comment:
Thanks for the update! Just a question, does this PR depend on lightning changes?